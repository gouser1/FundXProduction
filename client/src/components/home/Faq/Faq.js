import React from "react";
import { Grid, Box } from "@material-ui/core";
import Accordion from "@material-ui/core/Accordion";
import { Link, useHistory } from "react-router-dom";
import AccordionSummary from "@material-ui/core/AccordionSummary";
import AccordionDetails from "@material-ui/core/AccordionDetails";
import Typography from "@material-ui/core/Typography";
import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
import useStyles from "./FaqStyle";

function Faq(props) {
  const classes = useStyles();
  const history = useHistory();

  return (
    <div className={classes.root}>
      <Grid
        container
        spacing={0}
        direction="column"
        alignItems="center"
        justify="center"
      >
        <Grid item lg={6} md={6} sm={10} xs={12}>
          <Box textAlign="center" pt={10} pb={5}>
            <Typography className={classes.h1}>
              Frequently Asked Questions
            </Typography>
          </Box>
        </Grid>
        <Grid item lg={6} md={6} sm={10} xs={12}>
          <Box>
            <Accordion>
              <AccordionSummary
                expandIcon={<ExpandMoreIcon />}
                aria-controls="panel1a-content"
                id="panel1a-header"
              >
                <Typography className={classes.h2}>
                  Do I have to sign up to use FundX?
                </Typography>
              </AccordionSummary>
              <AccordionDetails>
                <Typography className={classes.p1}>
                  No, you can use our application as a guest user and still have
                  access to a range of our features. Guest users will be able to
                  see pitches and contact users.
                </Typography>
              </AccordionDetails>
            </Accordion>
            <Accordion>
              <AccordionSummary
                expandIcon={<ExpandMoreIcon />}
                aria-controls="panel2a-content"
                id="panel2a-header"
              >
                <Typography className={classes.h2}>What is FundX</Typography>
              </AccordionSummary>
              <AccordionDetails>
                <Typography className={classes.p1}>
                  FundX is an Angel-led Investment platform that connects high
                  grow potentional companies with experienced investors.
                </Typography>
              </AccordionDetails>
            </Accordion>
            <Accordion>
              <AccordionSummary
                expandIcon={<ExpandMoreIcon />}
                aria-controls="panel2a-content"
                id="panel2a-header"
              >
                <Typography className={classes.h2}>
                  What can I do with FundX?
                </Typography>
              </AccordionSummary>
              <AccordionDetails>
                <Typography className={classes.p1}>
                  FundX allows you to create or find pitches from companies
                  looking to grow and expand their business.
                </Typography>
              </AccordionDetails>
            </Accordion>
            <Accordion>
              <AccordionSummary
                expandIcon={<ExpandMoreIcon />}
                aria-controls="panel2a-content"
                id="panel2a-header"
              >
                <Typography className={classes.h2}>How do I start?</Typography>
              </AccordionSummary>
              <AccordionDetails>
                <Typography className={classes.p1}>
                  Sign up for free{" "}
                  <Link
                    onClick={() => {
                      history.push("/register");
                    }}
                  >
                    {" "}
                    here{" "}
                  </Link>
                </Typography>
              </AccordionDetails>
            </Accordion>
            <Accordion>
              <AccordionSummary
                expandIcon={<ExpandMoreIcon />}
                aria-controls="panel2a-content"
                id="panel2a-header"
              >
                <Typography className={classes.h2}>
                  How reliable is FundX?
                </Typography>
              </AccordionSummary>
              <AccordionDetails>
                <Typography className={classes.p1}>
                  We have an average uptime of 99,9%. Any system related issues
                  are reported and updated
                </Typography>
              </AccordionDetails>
            </Accordion>
            <Accordion>
              <AccordionSummary
                expandIcon={<ExpandMoreIcon />}
                aria-controls="panel2a-content"
                id="panel2a-header"
              >
                <Typography className={classes.h2}>
                  How does FundX ensure security?
                </Typography>
              </AccordionSummary>
              <AccordionDetails>
                <Typography className={classes.p1}>
                  FundX enforces a password complexity standard and credentials
                  are encrypted when stored in the database.
                </Typography>
              </AccordionDetails>
            </Accordion>

            <Accordion>
              <AccordionSummary
                expandIcon={<ExpandMoreIcon />}
                aria-controls="panel2a-content"
                id="panel2a-header"
              >
                <Typography className={classes.h2}>
                  How do I track pitches I am intersted in?
                </Typography>
              </AccordionSummary>
              <AccordionDetails>
                <Typography className={classes.p1}>
                  FundX has a favourite system for registered users so you can
                  easily keep track of them.
                </Typography>
              </AccordionDetails>
            </Accordion>
          </Box>
        </Grid>
      </Grid>
    </div>
  );
}

export default Faq;
