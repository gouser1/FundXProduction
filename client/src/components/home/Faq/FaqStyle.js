import { makeStyles } from "@material-ui/styles";

export default makeStyles(() => ({
  root: {
    flexGrow: 1,
    paddingBottom: "15%",
  },

  button: {
    fontFamily: "'Karla', sans-serif;",
    fontSize: "1.2em",
    color: "white",
    backgroundColor: "#3bc693",
    "&:hover": {
      backgroundColor: "#34b384",
    },
    textTransform: "none",
  },
  h1: {
    textAlign: "center",
    fontFamily: "'Karla', sans-serif;",
    fontSize: "1.5em",
    color: "#22303d",
    fontWeight: "700",
    "@media (min-width:700px)": {
      fontSize: "3em",
    },
  },
  h2: {
    fontFamily: "'Karla', sans-serif;",
    fontSize: "1.2em",
    color: "#46637f",
    fontWeight: "500",
    "@media (min-width:700px)": {
      fontSize: "1.2rem",
    },
  },

  p1: {
    textAlign: "left",
    fontFamily: "'Rubik', sans-serif;",
    fontSize: "1em",
    color: "#22303d",
    letterSpacing: "0.02em",
    lineHeight: "1.4em",
    "@media (max-width:700px)": {
      fontSize: "1rem",
      textAlign: "center",
    },
    "@media (max-width:1400px)": {
      fontSize: "1rem",
    },
  },
}));
