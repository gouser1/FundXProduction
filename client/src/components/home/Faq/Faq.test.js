import React from "react";
import renderer from "react-test-renderer";

import Faq from "./Faq";

it("renders correctly", () => {
  const tree = renderer.create(<Faq />).toJSON();
  expect(tree).toMatchSnapshot();
});
