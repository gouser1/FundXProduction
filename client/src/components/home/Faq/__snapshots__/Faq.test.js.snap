// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-content-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-10 MuiGrid-grid-md-6 MuiGrid-grid-lg-6"
    >
      <div
        className="MuiBox-root MuiBox-root-6"
      >
        <p
          className="MuiTypography-root makeStyles-h1-3 MuiTypography-body1"
        >
          Frequently Asked Questions
        </p>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-10 MuiGrid-grid-md-6 MuiGrid-grid-lg-6"
    >
      <div
        className="MuiBox-root MuiBox-root-7"
      >
        <div
          className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
        >
          <div
            aria-controls="panel1a-content"
            aria-disabled={false}
            aria-expanded={false}
            className="MuiButtonBase-root MuiAccordionSummary-root"
            id="panel1a-header"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiAccordionSummary-content"
            >
              <p
                className="MuiTypography-root makeStyles-h2-4 MuiTypography-body1"
              >
                Do I have to sign up to use FundX?
              </p>
            </div>
            <div
              aria-disabled={false}
              aria-hidden={true}
              className="MuiButtonBase-root MuiIconButton-root MuiAccordionSummary-expandIcon MuiIconButton-edgeEnd"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role={null}
              tabIndex={null}
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
            </div>
          </div>
          <div
            className="MuiCollapse-root MuiCollapse-hidden"
            style={
              Object {
                "minHeight": "0px",
              }
            }
          >
            <div
              className="MuiCollapse-wrapper"
            >
              <div
                className="MuiCollapse-wrapperInner"
              >
                <div
                  aria-labelledby="panel1a-header"
                  id="panel1a-content"
                  role="region"
                >
                  <div
                    className="MuiAccordionDetails-root"
                  >
                    <p
                      className="MuiTypography-root makeStyles-p1-5 MuiTypography-body1"
                    >
                      No, you can use our application as a guest user and still have access to a range of our features. Guest users will be able to see pitches and contact users.
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
        >
          <div
            aria-controls="panel2a-content"
            aria-disabled={false}
            aria-expanded={false}
            className="MuiButtonBase-root MuiAccordionSummary-root"
            id="panel2a-header"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiAccordionSummary-content"
            >
              <p
                className="MuiTypography-root makeStyles-h2-4 MuiTypography-body1"
              >
                What is FundX
              </p>
            </div>
            <div
              aria-disabled={false}
              aria-hidden={true}
              className="MuiButtonBase-root MuiIconButton-root MuiAccordionSummary-expandIcon MuiIconButton-edgeEnd"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role={null}
              tabIndex={null}
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
            </div>
          </div>
          <div
            className="MuiCollapse-root MuiCollapse-hidden"
            style={
              Object {
                "minHeight": "0px",
              }
            }
          >
            <div
              className="MuiCollapse-wrapper"
            >
              <div
                className="MuiCollapse-wrapperInner"
              >
                <div
                  aria-labelledby="panel2a-header"
                  id="panel2a-content"
                  role="region"
                >
                  <div
                    className="MuiAccordionDetails-root"
                  >
                    <p
                      className="MuiTypography-root makeStyles-p1-5 MuiTypography-body1"
                    >
                      FundX is an Angel-led Investment platform that connects high grow potentional companies with experienced investors.
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
        >
          <div
            aria-controls="panel2a-content"
            aria-disabled={false}
            aria-expanded={false}
            className="MuiButtonBase-root MuiAccordionSummary-root"
            id="panel2a-header"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiAccordionSummary-content"
            >
              <p
                className="MuiTypography-root makeStyles-h2-4 MuiTypography-body1"
              >
                What can I do with FundX?
              </p>
            </div>
            <div
              aria-disabled={false}
              aria-hidden={true}
              className="MuiButtonBase-root MuiIconButton-root MuiAccordionSummary-expandIcon MuiIconButton-edgeEnd"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role={null}
              tabIndex={null}
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
            </div>
          </div>
          <div
            className="MuiCollapse-root MuiCollapse-hidden"
            style={
              Object {
                "minHeight": "0px",
              }
            }
          >
            <div
              className="MuiCollapse-wrapper"
            >
              <div
                className="MuiCollapse-wrapperInner"
              >
                <div
                  aria-labelledby="panel2a-header"
                  id="panel2a-content"
                  role="region"
                >
                  <div
                    className="MuiAccordionDetails-root"
                  >
                    <p
                      className="MuiTypography-root makeStyles-p1-5 MuiTypography-body1"
                    >
                      FundX allows you to create or find pitches from companies looking to grow and expand their business.
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
        >
          <div
            aria-controls="panel2a-content"
            aria-disabled={false}
            aria-expanded={false}
            className="MuiButtonBase-root MuiAccordionSummary-root"
            id="panel2a-header"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiAccordionSummary-content"
            >
              <p
                className="MuiTypography-root makeStyles-h2-4 MuiTypography-body1"
              >
                How do I start?
              </p>
            </div>
            <div
              aria-disabled={false}
              aria-hidden={true}
              className="MuiButtonBase-root MuiIconButton-root MuiAccordionSummary-expandIcon MuiIconButton-edgeEnd"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role={null}
              tabIndex={null}
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
            </div>
          </div>
          <div
            className="MuiCollapse-root MuiCollapse-hidden"
            style={
              Object {
                "minHeight": "0px",
              }
            }
          >
            <div
              className="MuiCollapse-wrapper"
            >
              <div
                className="MuiCollapse-wrapperInner"
              >
                <div
                  aria-labelledby="panel2a-header"
                  id="panel2a-content"
                  role="region"
                >
                  <div
                    className="MuiAccordionDetails-root"
                  >
                    <p
                      className="MuiTypography-root makeStyles-p1-5 MuiTypography-body1"
                    >
                      Sign up for free
                       
                      <a
                        className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                        onBlur={[Function]}
                        onClick={[Function]}
                        onFocus={[Function]}
                      >
                         
                        here
                         
                      </a>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
        >
          <div
            aria-controls="panel2a-content"
            aria-disabled={false}
            aria-expanded={false}
            className="MuiButtonBase-root MuiAccordionSummary-root"
            id="panel2a-header"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiAccordionSummary-content"
            >
              <p
                className="MuiTypography-root makeStyles-h2-4 MuiTypography-body1"
              >
                How reliable is FundX?
              </p>
            </div>
            <div
              aria-disabled={false}
              aria-hidden={true}
              className="MuiButtonBase-root MuiIconButton-root MuiAccordionSummary-expandIcon MuiIconButton-edgeEnd"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role={null}
              tabIndex={null}
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
            </div>
          </div>
          <div
            className="MuiCollapse-root MuiCollapse-hidden"
            style={
              Object {
                "minHeight": "0px",
              }
            }
          >
            <div
              className="MuiCollapse-wrapper"
            >
              <div
                className="MuiCollapse-wrapperInner"
              >
                <div
                  aria-labelledby="panel2a-header"
                  id="panel2a-content"
                  role="region"
                >
                  <div
                    className="MuiAccordionDetails-root"
                  >
                    <p
                      className="MuiTypography-root makeStyles-p1-5 MuiTypography-body1"
                    >
                      We have an average uptime of 99,9%. Any system related issues are reported and updated
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
        >
          <div
            aria-controls="panel2a-content"
            aria-disabled={false}
            aria-expanded={false}
            className="MuiButtonBase-root MuiAccordionSummary-root"
            id="panel2a-header"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiAccordionSummary-content"
            >
              <p
                className="MuiTypography-root makeStyles-h2-4 MuiTypography-body1"
              >
                How does FundX ensure security?
              </p>
            </div>
            <div
              aria-disabled={false}
              aria-hidden={true}
              className="MuiButtonBase-root MuiIconButton-root MuiAccordionSummary-expandIcon MuiIconButton-edgeEnd"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role={null}
              tabIndex={null}
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
            </div>
          </div>
          <div
            className="MuiCollapse-root MuiCollapse-hidden"
            style={
              Object {
                "minHeight": "0px",
              }
            }
          >
            <div
              className="MuiCollapse-wrapper"
            >
              <div
                className="MuiCollapse-wrapperInner"
              >
                <div
                  aria-labelledby="panel2a-header"
                  id="panel2a-content"
                  role="region"
                >
                  <div
                    className="MuiAccordionDetails-root"
                  >
                    <p
                      className="MuiTypography-root makeStyles-p1-5 MuiTypography-body1"
                    >
                      FundX enforces a password complexity standard and credentials are encrypted when stored in the database.
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
        >
          <div
            aria-controls="panel2a-content"
            aria-disabled={false}
            aria-expanded={false}
            className="MuiButtonBase-root MuiAccordionSummary-root"
            id="panel2a-header"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiAccordionSummary-content"
            >
              <p
                className="MuiTypography-root makeStyles-h2-4 MuiTypography-body1"
              >
                How do I track pitches I am intersted in?
              </p>
            </div>
            <div
              aria-disabled={false}
              aria-hidden={true}
              className="MuiButtonBase-root MuiIconButton-root MuiAccordionSummary-expandIcon MuiIconButton-edgeEnd"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role={null}
              tabIndex={null}
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
            </div>
          </div>
          <div
            className="MuiCollapse-root MuiCollapse-hidden"
            style={
              Object {
                "minHeight": "0px",
              }
            }
          >
            <div
              className="MuiCollapse-wrapper"
            >
              <div
                className="MuiCollapse-wrapperInner"
              >
                <div
                  aria-labelledby="panel2a-header"
                  id="panel2a-content"
                  role="region"
                >
                  <div
                    className="MuiAccordionDetails-root"
                  >
                    <p
                      className="MuiTypography-root makeStyles-p1-5 MuiTypography-body1"
                    >
                      FundX has a favourite system for registered users so you can easily keep track of them.
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
